build/build.proj

```diff
@@ -37,7 +37,7 @@
     <PropertyGroup>
       <TestProjectPaths>@(CoreFuncTestProjects)</TestProjectPaths>
       <TestResultsFileName>CoreFuncTests</TestResultsFileName>
-      <TestVisualStudioVersion>15.0</TestVisualStudioVersion>
```

Comment:
I'm not sure there's a need for TestVisualStudioVersion in the first place since we literally just `VisualStudioVersion=$(TestVisualStudioVersion);` below.

Reply:
When you don't set a VisualStudioVersion it's the MSBuild version used to build it dictates it.

Reply:
Removed.